user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -1927,6 +1933,10 @@ define([
     return response;
   }
 
+  function _isRepositoryPath(path) {
+    return path.charAt(0) === "/" || path.charAt(0) === ":";
```

Comment:
Could all calls to this method be made with an unencoded path? I think so, but may be wrong.
Not great to be spreading details of the URL-path encoding logic of generic paths.

Reply:
Yeah I can move the check up to the top of `[updateFolderButtons](https://github.com/pentaho/pentaho-platform/blame/9c4ee216f720cac22e03e29a84ae7180aec4aa12/user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js#L341)` to before it encodes anything. The encoding logic in that function is... weird.